cohttp-eio/src/cohttp_eio.mli

```diff
@@ -96,3 +100,94 @@ module Server : sig
 
   val not_found_handler : handler
 end
+
+module Client : sig
+  type response = Http.Response.t * Eio.Buf_read.t
+
+  type host = string * int option
+  (** Represents a server host domain name and port, e.g. www.example.org:8080,
+      www.reddit.com *)
+
+  type resource_path = string
+  (** Represents HTTP request resource path, e.g. "/shop/purchase",
+      "/shop/items", "/shop/categories/" etc. *)
+
+  type 'a conn = unit -> (host * Eio.Flow.two_way as 'a)
```

Comment:
This type isn't actually polymorphic as written. Also, does it need to be a function? It looks like everything that takes it just calls the function once immediately at the start.
```suggestion
  type 'a conn = host * (#Eio.Flow.two_way as 'a)
```

Reply:
This is done now.